azure/datalake/store/lib.py

```diff
@@ -97,10 +103,17 @@ def auth(tenant_id=default_tenant, username=default_username,
     context = adal.AuthenticationContext('https://login.microsoftonline.com/' +
```

Comment:
This means Azure US only, correct? no usage on mooncake or somewhere for authentication?